src/master-playlist-controller.js

```diff
@@ -808,8 +808,8 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     if (this.mediaTypes_.AUDIO.activePlaylistLoader) {
       // if the audio playlist loader exists, then alternate audio is active
-      if (!this.mainSegmentLoader_.startingMedia_ ||
-          this.mainSegmentLoader_.startingMedia_.hasVideo) {
+      if (!this.mainSegmentLoader_.startingMediaInfo_ ||
```

Comment:
I think we want to using `startingMediaInfo` here as we want to call end of stream regardless of the state of `currentMediaInfo`. This is how it worked prior to the codec switching changes. It's possible that this isn't an issue though and we should just keep it as `currentMediaInfo`